### PR TITLE
Disabled multiview for firetv

### PR DESCRIPTION
**Pull request by olbapbetan**

### CODE REVIEW

File: Viewer.js [hunk 1]
```diff
@@ -337,7 +337,8 @@ class MillicastWidget extends React.Component {
                   {this.state.playing ? 'Pause' : 'Play'}
                 </Text>
               </TouchableHighlight>
-              {this.state.playing ? (
+              {this.state.playing &&
+              !JSON.stringify(Platform).includes('AFTMM') ? (
```
Comment (Santiago-Souto) [inline Viewer.js:341]: There are some other FireTV models that start with `AFT` but not `AFTMM`, it should include all possible values of FireTV.
Also, AndroidTV seems to be having the same issues, could it be disabled for that as well?

**Review by Santiago-Souto — CHANGES_REQUESTED** [review]

File: Viewer.js [hunk 2]
```diff
@@ -239,6 +239,7 @@ class MillicastWidget extends React.Component {
   };
 
   render() {
+    console.log(Platform.Brand, 'Platform.Brand');
```
[the remark this answers is not in the document]
Comment (olbapbetan) [reply Viewer.js]: It was already removed.

File: Viewer.js (hunk 1 above)
Comment (olbapbetan) [reply Viewer.js:341]: Disabling this button for all AndroidTV-based devices (such as FireTV) would require a different approach.

**Comment by Santiago-Souto** [comment]
This PR was replaced by #90

**Pull request closed without merging**